packages/wrangler/src/index.ts

```diff
@@ -271,7 +271,7 @@ export function createCLIParser(argv: string[]) {
 
 	// docs
 	wrangler.command(
-		"docs [command]",
+		"docs [command..]",
```

Comment:
@GregBrimble I figured it out - this lets you have an optional command that takes an array

so `wrangler docs pages astro` works now

Reply:
or maybe it was @WalshyDev, forgot which one I showed this to lol

Reply:
Oh hell yes. Ty Max!